i6_models/assemblies/e_branchformer/e_branchformer_v1.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import annotations
+
+__all__ = [
+    "EbranchformerBlockV1Config",
+    "EbranchformerBlockV1",
+    "EbranchformerEncoderV1Config",
+    "EbranchformerrEncoderV1",
```

Comment:
I think there is a Typo in the whole module, I will just mark here, but there are a few lines affected.